src/open3e/Open3Eenums.py

```diff
@@ -2328,4 +2328,58 @@
         2: "Installed",
         3: "Factory Installed",
     },
-}
+
+#### NEW NEW NEW
+    #erledigt
+    "Roomtypes" : {
+        2: "Bathroom", #Badezimmer
+        3: "Bedroom", #Schlafzimmer
+        4: "Hallway", #Diele/Flur
+        5: "Living Room", #Wohnzimmer
+        6: "Childrens Room", #Kinderzimmer
+        7: "Kitchen", #Küche
+        8: "Office", #Büro
+        9: "Guest Toilet", #WCToilette
+        11: "Others" #Andere
+    },
+    
+    #erledigt
+    "TemperatureControlLevels" : {
+        1: "MaximumEco",
+        2: "MoreEco",
+        3: "Eco",
+        4: "Komfort (Default)",
+        5: "More Comfort",
+        6: "Maximum Comfort"
+    },
+
+    #erledigt
+    "WindowDetectionState" : {
+        0: "Not Active",
+        1: "Active",
+    },
+
+    #erledigt
+    "InstalledValveType" : {
+        4: "powerless closed",
+        5: "powerless opened",
+    },
```

Comment:
Eventuell macht auch eine Umbenennung von `InstalledValveType` zu `DeviceType` sinn?

```suggestion
    "InstalledValveType" : {
        1: "Climate sensor",
        2: "Thermostatic radiator valve",
        4: "powerless closed",
        5: "powerless opened",
        7: "Repeater"
    },
```

Reply:
@MyHomeMyData  das könnten wir ja eigentlich 'gefahrlos' in's develop übernehmen, oder?

Reply:
@philippoo66 Die enums können wir natürlich erweitern. Aber das hat keinen Effekt, wenn die Datenpunktdefinition nicht ebenfalls angepasst wird, oder? Und da haben wir weiterhin keine Lösung für die unterschiedlichen Längen.

Reply:
die "Lösung" ist ja bislang die Wiki Seite mit den divergenten Codecs...

ich möchte halt nur vermeiden, dass irgendwelche mühsam errungenen Erkenntnisse untergehen und man sie in den Discussions suchen muss und wohlmöglich kaum wiederfindet....

also **meine Bitte an alle und @Backes89  und @reey die DID Codecs, die wir so nicht unbedingt übernehmen können,** weil die Länge von der in der datapoints.py abbweicht, **in's Wiki zu pasten auf https://github.com/open3e/open3e/wiki/070-%E2%80%90-List-of-divergent-codecs**  (und da ggf auch upzudaten)! Das Wiki ist offen, da kann jeder Konstruktives beitragen...

und die ValveTypes nehm ich mal grad in's develop enums.py - schaden tut es ja da auf keinen Fall, und ggf wird es ja referenziert

Reply:
> Eventuell macht auch eine Umbenennung von `InstalledValveType` zu `DeviceType` sinn?

Ich denke hier kann man 'ViCareDeviceType' raus machen, da die eNum sich auf die ViCare Geräte bezieht. 

powerless closed/open könnte man in 
4: "actuator underfloor heating - powerless closed",
5: "actuator underfloor heating - powerless opened",
abändern.

<img width="507" height="154" alt="image" src="https://github.com/user-attachments/assets/61e33ded-d381-4527-b1df-65bcdfba8b96" />

Reply:
Passe ich alles mal an und packe das hier in der PR rein. Wird denke Montag werden.